package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hyper-content-for",
+  "version": "0.1.0",
+  "description": "Allows you to store HTML for use (and re-use) later.",
+  "main": "hyper-content-for.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperoslo/hyper-content-for-angular.git"
+  },
+  "keywords": [
+    "hyper",
+    "angular",
+    "content",
+    "yield",
+    "template",
+    "layout"
+  ],
+  "author": "Sindre Moen <sindrenm@gmail.com>",
```

Comment:
It could be, but in our gems we've actually used the author of the gem:

https://github.com/hyperoslo/gamification/blob/master/gamification.gemspec#L10

This is also the way it's configured in the bower.json file, so I'd stick with it for now and perhaps change it later.